invenio_rdm_records/fixtures/users.py

```diff
@@ -22,20 +23,32 @@
 class UsersFixture:
     """Users fixture."""
 
-    def __init__(self, search_path, filename):
+    def __init__(self, search_paths, filename):
         """Initialize the fixture."""
-        self._search_path = search_path
+        self._search_paths = search_paths
         self._filename = filename
 
     def load(self):
-        """Load the fixture."""
-        with open(self._search_path.path(self._filename)) as fp:
-            data = yaml.load(fp)
-            for email, user_data in data.items():
-                self.create_user(email, user_data)
+        """Load the fixture.
 
-    def create_user(self, email, entry):
-        """Load a single user."""
+        Fixtures found sooner in self._search_paths have priority.
+        """
+        for path in self._search_paths:
+            filepath = path / self._filename
+
+            # Providing a users.yaml file is optional
+            if not filepath.exists():
+                continue
+
+            with open(filepath) as fp:
+                data = yaml.safe_load(fp) or {}
+                for email, user_data in data.items():
+                    self.create_user(email, user_data)
+
+            break
```

Comment:
we `break` becuase there cannot be more than one *users* file? maybe it can be explained in a comment or in the docstring.

Reply:
Clarified this in the function description :+1: